src/gui/symbology/qgscategorizedsymbolrendererwidget.cpp

```diff
@@ -250,6 +251,12 @@ QVariant QgsCategorizedSymbolRendererModel::data( const QModelIndex &index, int
       }
       break;
     }
+    case Qt::UserRole:
```

Comment:
Would be nicer to make this an enum of CustomRoles (in qgscategorizedsymbolrendererwidget.h), and then refer to it using the enum instead of the raw "userrole" value. It makes the code more readable and also avoids potential future issues if someone decides to add another user role to the model.